src/Packlist/PacklistForm.js

```diff
@@ -0,0 +1,218 @@
+import React, { useState, useRef } from 'react'
+import { useForm } from 'react-hook-form'
+import { v4 as uuid } from 'uuid'
+import styled from 'styled-components/macro'
+import Input from '../common/Input'
+import Label from '../common/Label'
+import Checkbox from '../common/Checkbox'
+import ErrorMessage from '../common/ErrorMessage'
+import PropTypes from 'prop-types'
+import ListContainer from '../common/ListContainer'
+import ListItem from '../common/ListItem'
+import usePacklists from './usePacklists'
+
+PacklistForm.propTypes = {
+  onPacklistSave: PropTypes.func.isRequired,
+}
+
+export default function PacklistForm({ onPacklistSave }) {
+  const { register, handleSubmit, reset, errors } = useForm()
+  const [items, setItems] = useState([])
+  const { packlists } = usePacklists()
+  const itemRef = useRef(null)
+  const onSubmit = (packlist, event) => {
+    event.preventDefault()
+    // for testing...
```

Comment:
this will be a merged in production branch. so why "for testing..." ?

Reply:
this is needed for testing, otherwise the test doesn't find this function. it's necessary to put it in this file